src/librustc_typeck/check/method/confirm.rs

```diff
@@ -463,24 +448,10 @@ impl<'a, 'gcx, 'tcx> ConfirmContext<'a, 'gcx, 'tcx> {
     ///////////////////////////////////////////////////////////////////////////
     // RECONCILIATION
 
-    /// When we select a method with an `&mut self` receiver, we have to go convert any
+    /// When we select a method with a mutable autoref, we have to go convert any
```

Comment:
Nit: I think this comment is not as good as the one before, even if it is potentially more precise (at least in the future...). Including concrete Rust code examples is always helpful, particularly as our terminology tends to drift over time. :) "a mutable autoref (e.g., `&mut self`)" would be better, for example.